driver/random.hpp

```diff
@@ -2,15 +2,18 @@
 #define GUARD_RANDOM_GEN_
 
 #include <cstdlib>
+#include <ctime>
+#include <cstdint>
 
 template <typename T>
 inline T FRAND()
 {
-    double d = static_cast<double>(rand() / (static_cast<double>(RAND_MAX)));
+    std::uint32_t seed = time(nullptr);
```

Comment:
I would not recommend using `std::time()` for seed since it returns time in seconds and consecutive calls will return the same value - this mean we will have the same "random" value during each second.

I guess `std::chrono::steady_clock` or `std::chrono::high_resolution_clock` is better choice since usually it has nanosecond resolution (and usually they are the same).

Reply:
Thanks for your suggestion. I updated the code accordingly.